tests/unit/src/Internal/Payment/State/ProcessedStateTest.php

```diff
@@ -93,8 +101,24 @@ public function test_complete_processing_will_transition_to_completed_state() {
 			->with( CompletedState::class, $this->mock_context )
 			->willReturn( $mock_completed_state );
 
+		$this->mock_legacy_proxy
+			->expects( $this->once() )
+			->method( 'call_function' )
+			->with( 'wc' )
+			->willReturnCallback(
+				function () {
+					$mock_cart = $this->getMockBuilder( \stdClass::class )
+						->setMethods( [ 'empty_cart' ] )
```

Comment:
`setMethods` is deprecard in favor of `onlyMethods`.
```suggestion
						->onlyMethods( [ 'empty_cart' ] )
```

Reply:
used `addMethods`